docs/setup/operation/configuration.md

```diff
@@ -281,6 +269,12 @@ If both are defined, then the **environment variables** will take priority.
     <td>http://repo1.maven.org/maven2/</td>
     <td>Remote principal repository for interpreter's additional dependency loading</td>
   </tr>
+  <tr>
+    <td><h6 class="properties">ZEPPELIN_INTERPRETER_MAX_POOL_SIZE</h6></td>
+    <td><h6 class="properties">zeppelin.interpreter.max.poolsize</h6></td>
+    <td>10</td>
+    <td>The maximum number of paragraphs that will be run at cron.</td>
```

Comment:
This doesn't sound like a correct description. `zeppelin.interpreter.max.poolsize` is for the number threads of per session configuration. That means if you are using 20 different interpreters, they can run simultaneously without affecting each other.

Reply:
I took note with 4 jdbc paragraph and a lof of python paragraphs.
You can see that only first 10 paragraphs were done.
![first10](https://user-images.githubusercontent.com/9324163/35802242-5cd9a258-0a80-11e8-9c94-ae8238b8893f.gif)

Reply:
@mebelousov This might be fixed in ZEPPELIN-3171, could you try that ?

Reply:
Year! This is fixed.
@zjffdu thank you!